Title: Window Title Environment Variable
Author: Eric Pruitt (https://www.codevat.com/, https://github.com/ericpruitt/)

This patch sets the environment variable "DWM_SELECTED_WINDOW_TITLE" to the
title of the currently selected window when spawning a subprocess.

diff --git a/dwm.c b/dwm.c
index 421bf27..810e4dc 100644
--- a/dwm.c
+++ b/dwm.c
@@ -1639,2 +1639,5 @@ spawn(const Arg *arg)
 		setsid();
+		if (selmon->sel) {
+			setenv("DWM_SELECTED_WINDOW_TITLE", selmon->sel->name, 1);
+		}
 		execvp(((char **)arg->v)[0], (char **)arg->v);
